Events.php

```diff
@@ -244,4 +245,23 @@ public static function onSpaceSidebarInit($event)
         }
     }
 
+    public static function onStreamAfterApplyFilters($event)
+    {
+        try {
```

Comment:
As far as I understood this is only required for already existing pages, can't we rather add a migration for this instead of intercepting the stream?

Reply:
Also maybe add a check in `beforeSave` to force the visiblity in this case

Reply:
yes, this is for existing pages, ok I will replace this with new migration, and will create additional check in `beforeSave`.

Reply:
done https://github.com/humhub/humhub-modules-custom-pages/pull/157/commits/15a48bb58423ca4b0c424ef986e0f0df1a5dc931